code/modules/food_and_drinks/plate.dm

```diff
@@ -0,0 +1,63 @@
+/obj/item/plate
```

Comment:
shouldn't these be subtypes of /obj/item/kitchen? or are you worried about the terrifying possibility of chefs storing plates in the suit storage slots of their aprons?

Reply:
that sounds like a dumb subtype.

Reply:
but it's one that exists nonetheless.